homeassistant/components/plugwise/sensor.py

```diff
@@ -303,8 +303,8 @@
     SensorEntityDescription(
         key="gas_consumed_interval",
         name="Gas consumed interval",
-        native_unit_of_measurement=UnitOfVolume.CUBIC_METERS,
-        device_class=SensorDeviceClass.GAS,
+        icon="mdi:meter-gas",
+        native_unit_of_measurement="m³/h",
         state_class=SensorStateClass.TOTAL,
```

Comment:
As the sensor becomes a rate, it can't be a total.

Reply:
Ok, but that would mean that all the other `electricity_xyz_interval` sensors also cannot have the state_class TOTAL?

Reply:
![image](https://user-images.githubusercontent.com/11290930/217048719-13d59316-d204-4bb0-9ee1-8c5eb8b37aef.png)

Reply:
They aren't marked as `kWh/h` but as `kWh`, which defines an accumulated total, not a Flowrate as this change contains. Flow rate are a measurement, not a total. You currently given unit to this sensor doesn't match the state class.

Reply:
Ok, I need to think about this a bit more. Other stuff is taking up brain-space...
Anyway, what is your suggestion for the state_class for the `gas_consumed_interval` sensor?
None? Or MEASUREMENT?

Reply:
Changed the state_class to MEASUREMENT, as suggested :)